blockchain/chain.go

```diff
@@ -2275,6 +2314,14 @@ type Config struct {
 	IndexManager IndexManager
 }
 
+func (b *BlockChain) initializeStakeVersionCaches() {
```

Comment:
Please move this to the test code since that's the only thing that really should be using it.  The `BlockChain` struct instance should be making the maps in the definition to be consistent with all the other maps and it's only in a single location.